codespell_lib/data/dictionary.txt

```diff
@@ -19897,6 +19902,7 @@ quarentine->quarantine
 quartically->quadratically
 quatation->quotation
 quater->quarter
+quating->quoting
```

Comment:
Suggest squatting

Reply:
Also a lot less likely in code, but will do.

Reply:
Done.